services/twilio-mock-service/src/main.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  BadRequestException,
+  HttpException,
+  HttpStatus,
+  ValidationPipe,
+} from '@nestjs/common';
+import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
+import * as bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import { ApplicationModule } from './app.module';
+import {
+  APP_FAVICON,
+  APP_TITLE,
+  APP_VERSION,
+  PORT,
+  ROOT_URL,
+  SWAGGER_CUSTOM_CSS,
+  SWAGGER_CUSTOM_JS,
+} from './config';
+
+async function bootstrap(): Promise<void> {
+  const app = await NestFactory.create(ApplicationModule);
+
+  const expressInstance = app.getHttpAdapter().getInstance();
+
+  expressInstance.disable('x-powered-by');
+
+  app.setGlobalPrefix('api');
+
+  const options = new DocumentBuilder()
+    .setTitle(APP_TITLE)
+    .setVersion(APP_VERSION)
+    .addServer(ROOT_URL)
+    .build();
+  const document = SwaggerModule.createDocument(app, options);
+  SwaggerModule.setup('/docs', app, document, {
+    customSiteTitle: APP_TITLE,
+    customfavIcon: APP_FAVICON,
+    customCss: SWAGGER_CUSTOM_CSS,
+    customJs: `data:text/javascript;base64,${Buffer.from(
+      SWAGGER_CUSTOM_JS,
+    ).toString('base64url')}`,
+    swaggerOptions: {
+      persistAuthorization: true,
+      tagsSorter: 'alpha',
+      operationsSorter: 'alpha',
+      defaultModelExpandDepth: 10,
+    },
+  });
```

Comment:
See comment on `config.ts`.

Reply:
I am leaving most of the swagger layout thing. May be nice to use when we deploy it to an azure app service